### DISCUSSION TITLE
Xiaomi Smoke Detector  JTYJ-GD-01LM/BB

### DISCUSSION THREAD

**Discussion by saveriospeziali** [Koenkk/zigbee2mqtt#8438] [Devices]
## Information about the device + link
JTYJ-GD-01LM/BB

Hi Just arrived from Aliexpress an **JTYJ-GD-01LM/BB** instead of JTYJ-GD-01LM/BW

It seem it is the last Xiaomi Version 

I am unable to pair, I suspect it is no ZigBee

Does anyone know this new device?

Regards

**Comment by z-x-f** [comment]
May be bluetooth.

**Reply by saveriospeziali** [reply]
I think you are right, the problem is that the supplier sent me this version also if in the description there was JTYJ-GD-01LM/BW and now they refuse to refund (because they say "this is the current version", then I suggest to check before to buy

**Comment by nstrochkov** [comment]
I see this error when I am trying send selftest to device Xiaomi JTYJ-GD-01LM/BW:

Error Publish 'set' 'selftest' to '0x00158d00039dab30' failed: 'Error: Write 0x00158d00039dab30/1 ssIasZone({"65521":{"value":50397184,"type":35}}, {"sendWhenActive":false,"timeout":35000,"disableResponse":false,"disableRecovery":false,"disableDefaultResponse":true,"direction":0,"srcEndpoint":null,"reservedBits":0,"manufacturerCode":4447,"transactionSequenceNumber":null,"writeUndiv":false}) failed (Timeout - 62677 - 1 - 104 - 1280 - 4 after 35000ms)'
